src/collision/CollisionManager.js

```diff
@@ -100,8 +97,11 @@ exports = Class(function(supr) {
   this.start = function(collisionCheckID) {};
 
   this.removeCollisionsContaining = function(collider) {
-    if (this._collidersToRemove.indexOf(collider) === -1 ) {
-      this._collidersToRemove.push(collider);
+    if (this._checksToRemove.indexOf(collider) === -1) {
+      var checksOnCollider = this._colliderMap.getCollisionChecksOn(collider);
+      if (checksOnCollider) {
+        this._checksToRemove.push(checksOnCollider);
```

Comment:
You'll probably want to make sure the _checksToRemove array doesn't already contain the check before adding it.